src/app/public/modules/list-view-grid/list-view-grid.component.ts

```diff
@@ -279,6 +342,26 @@ export class SkyListViewGridComponent
     if (this.enableMultiselect) {
       this.dispatcher.toolbarShowMultiselectToolbar(true);
     }
+
+    if (this.messageStream) {
+      this.messageStream.subscribe((message: SkyListViewGridMessage) => {
```

Comment:
Looks like this code is duplicated from the setter, above? Any way to consolidate?

Reply:
Done.